host/frontend/liboperator/operator/operator.go

```diff
@@ -28,11 +28,20 @@ import (
 	"github.com/gorilla/mux"
 )
 
+const (
+	DefaultSocketPath     = "/run/cuttlefish/operator"
```

Comment:
I think is clearer if the binaries define their own default values. the problem I see here is that the library's users do not need to to use these defaults to use the library and the library does not use these values either so the library is exposing and being responsible of something that's not necessary needed.

Reply:
done